auditlog/event.go

```diff
@@ -199,31 +200,34 @@ func (e Event) addMetadata(key string, value interface{}) error {
 }
 
 // Publish delivers the event to WorkOS asyncronously.
-func (e Event) Publish() chan error {
+func (e Event) Publish() (chan error, chan *Event) {
```

Comment:
Errors are expected to be the second return value:

```suggestion
func (e Event) Publish() (chan *Event, chan error) {
```